### ISSUE TITLE
Search filter with multiple queries from meta fields

### ISSUE THREAD

**Issue by TheBigBanik** [Crocoblock/suggestions#1322]
I think it would be very helpful if we can search in multiple meta fields. For example, for a real estate webasite, users must be able to type and search both for cities, neighborhoods, Post code and other custom fields to avoid multiple filters.

**Comment by secondsky** [comment]
Indeed! Unfortunately I did see already multiple such wishes, including my own.
The developers seem to ignore it.

**Comment by aalexandr21** [comment]
JetSearch has a field where you can set any meta-field names
<img width="424" height="609" alt="Image" src="https://github.com/user-attachments/assets/28634c31-357d-4cf0-8e1c-4894731c4e48" />

If issue with JetSmartFilters, you can also paste any meta-field names there
<img width="1386" height="645" alt="Image" src="https://github.com/user-attachments/assets/8c1870fa-2d02-41f6-aa9c-c1c176873689" />
